### ISSUE TITLE
ERROR Key when I try synthesize

### ISSUE THREAD

**Issue by luis-vera** [mozilla/TTS#656]
Hi. I try MozillaTTS in a PC and I didn't have problems to synthesize some audios. I try MozillaTTS in an other PC but now when i try to synthesize It wasn't possible, but the message was:

key already registered with the same priority: GroupS partial Softmax

What can I do to solve this problem?
Thanks a lot

**Comment by erogol** [comment]
I don't think it is about TTS but Pytorch installation.
